tests/config_test.py

```diff
@@ -120,6 +120,24 @@ def test_parse_yaml_document_to_dict_missing_required_key(self):
             config.parse_yaml_document_to_dict,
             yaml_doc)
 
+    def test_load_from_dict(self):
+        config_data = {
+              'developer_token': self.developer_token,
+              'client_id': self.client_id,
+              'client_secret': self.client_secret,
+              'refresh_token': self.refresh_token
+          }
+        self.assertEqual(config.load_from_dict(config_data), config_data)
+    
+
+    def test_load_from_dict_error(self):
+        config_data = 111
+        self.assertRaises(
+        ValueError,
+        config.load_from_dict,
+        config_data
```

Comment:
Indent lines 136, 137, and 138 by four spaces and bring the below parenthesis up to line 138:

```python
self.assertRaises(
    ValueError,
    config.load_from_dict,
    config_data)
```